tls/s2n_server_new_session_ticket.c

```diff
@@ -190,21 +193,31 @@ S2N_RESULT s2n_tls13_server_nst_send(struct s2n_connection *conn, s2n_blocked_st
  *# unsigned integer in network byte order from the time of ticket
  *# issuance. 
  **/
-static S2N_RESULT s2n_generate_ticket_lifetime(struct s2n_connection *conn, uint32_t *ticket_lifetime)
+static S2N_RESULT s2n_generate_ticket_lifetime(struct s2n_connection *conn, uint64_t key_intro_time, uint64_t current_time, uint32_t *ticket_lifetime)
 {
```

Comment:
Note for this entire function, if you're doing any type of subtraction, you should assert you're not underflowing before actually doing the calculation.